packages/jobs/lib/crons/autoIdleDemo.integration.test.ts

```diff
@@ -33,32 +33,31 @@ describe('Auto Idle Demo', () => {
         });
 
         const connName = nanoid();
-        const env = await environmentService.createEnvironment(0, envName);
         await configService.createProviderConfig({
             unique_key: DEMO_GITHUB_CONFIG_KEY,
             provider: 'github',
-            environment_id: env!.id,
```

Comment:
unrelated to track_deletes: making environment id not optional, same from sync and syncJob below